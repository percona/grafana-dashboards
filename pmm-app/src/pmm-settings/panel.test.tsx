import React from 'react';
import { shallow } from 'enzyme';
import SettingsPanel from './panel';

jest.mock('shared/components/helpers/notification-manager');

// TODO: add correct shallow renderer, we need to have something to render to 2nd level
describe('Settings panel test', () => {
  it('Settings panel renders correct without props', () => {
    const root = shallow(<SettingsPanel />);

    expect(root).toMatchSnapshot();

    root.unmount();
  });
});
