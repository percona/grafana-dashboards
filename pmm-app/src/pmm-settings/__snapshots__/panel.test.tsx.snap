// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings panel test Settings panel renders correct without props 1`] = `
<div
  id="antd"
>
  <div
    className="app-theme-dark pmm-settings-panel"
  >
    <ReactFinalForm
      onSubmit={[Function]}
      render={[Function]}
    />
  </div>
</div>
`;
